apache/zeppelin
# [ZEPPELIN-1060] validate user before saving

### What is this PR for?
Zeppelin notebook permissions change has an issue while reconfiguring permissions by clearing textbox

### What type of PR is it?
[Bug Fix]

### Todos
* [x] - Validate user list before saving
* [x] - getPermissions from server again before showing popup

### What is the Jira issue?
* [ZEPPELIN-1060](https://issues.apache.org/jira/browse/ZEPPELIN-1060)

### How should this be tested?
1) I log in as user 'admin'
2) I create a notebook 'Untitled Notebbok 1' as user admin
3) I setup permissions of the notebook as owners = admin, readers= admin, writers= admin and click save button
It works file
4) Now I again try to give owners permission to 'All users' by clearing owners field. Since in step-3 owners = admin ; it allows me to save these changes successfully
5) Now I again try to toggle permissions of the notebook. Since in step-4 I cleared owners field, it should allow all users to change permissions. But it is not happening, instead it gives an error box that only User = [] has sufficient permissions


### Questions:
* Does the licenses files need update? n/a
* Is there breaking changes for older versions? n/a
* Does this needs documentation? n/a
